README.md

```diff
@@ -61,7 +56,13 @@ Execute the following command to uninstall Studio from your environment:
 $ helm uninstall studio --namespace studio
 ```
 
-## Parameters
+## Chart Configuration
+
+To further configure the chart values, you might want to create a `values.yaml` 
```

Comment:
> you might want to create a `values.yaml`

We need to rephrase this because users always need to create their own `values.yaml` to set some variables.

I was thinking of showcasing the chart configuration using examples for each SCM provider. However, it might be in the form of links to pages on our docs website.

Reply:
Also, `values.yaml` versus `override.yaml` - what's the most common filename for overriding the chart's own `values.yaml`?

Reply:
@jesper7  - you're right, I put this in draft exactly because the only-1-value instruction is basically toy. there's always going to be more. so I'm thinking of expanding this to:
- 1 liner installation case (like today)
- more realistic installation case, showing an example `values.yaml` that is fitting for a simple installation with ingresses.

`values.yaml` vs `override.yaml` - more familiar with `values.yaml` (or `<x>_values.yaml`/`<x>-values.yaml` when dealing with multiple installations) personally but whatever you prefer. 
short google led me [here](https://all.docs.genesys.com/PrivateEdition/Current/PEGuide/HelmOverrides) where they use `<x>_values.yaml` but it's not authoritative.
I don't feel strongly about this 🤷